src/widgets/index/index.ts

```diff
@@ -394,6 +394,10 @@ const index = (props: IndexProps): Index => {
             const requiredWidgets: Array<Widget['$$type']> =
               stateToWidgetsMap[parameter];
```

Comment:
a separate solution  could be 
```suggestion
              stateToWidgetsMap[parameter] || [];
```

this logic still works for unknown ui state

Reply:
the TS annotation is lying here, since it can also be undefined